acceptance/helper.rb

```diff
@@ -350,7 +350,7 @@ class { 'puppetdb::master::config':
       puppetdb_startup_timeout => 120,
       manage_report_processor  => true,
       enable_reports           => true,
-      strict_validation        => false,
+      strict_validation        => true,
```

Comment:
it's supplied to the connection validator in the module
https://github.com/puppetlabs/puppetlabs-puppetdb/blob/master/manifests/master/config.pp#L37-37